README.md

```diff
@@ -43,12 +43,16 @@ php_fpm_pools:
     pm_max_children: 20
     pm_start_servers: 20
     processes_priority: -19
-php_fpm_php_ini_values: 
-  apc.enabled:
-    - 1
-  extension:
-    - php_mysqli.dll
-    - php_ldap.dll
+php_fpm_php_ini_values:
```

Comment:
Nice, but have you also thought about modifying the defaults? Feels like #13 is relevant again

Reply:
Also, now that the variable is getting more complex, why not document it in the README?